### ISSUE TITLE
It does not print the log by console when I have a file to save it

### ISSUE THREAD

**Issue by frbuceta** [starak/node-console-stamp#40]
# **I make a small introduction to the problem**

I have version 3 of `console-stamp` and I want to save a log file by date.

When I have everything configured ... I see that it is saved in the file (incorrectly ... I explain now why) but it does not print by console. Ideally, the two would work at the same time and the log level could be configured for each of them.

Now I explain the problem that is inside the log file ... I have `console-stamp` with colors and that is what is saving me in the log

# **Down here a few pieces of my code**
**-> Console Output**
```
Starting Script
Environment is 'development'
Connecting to database...
Timeout: 5000
Configure connection...
```

**-> Log File**
`[35m[24.03.2020 13:49:51][39m [34m[INFO] [39m [35m[24.03.2020 13:49:51][39m [34m[INFO] [39m [35m[24.03.2020 13:49:51][39m
`

**-> Code**
```
const fs = require('fs');
require('console-stamp')(console, {
  format: ':date(dd.mm.yyyy HH:MM:ss, true).magenta :label(7)',
  tokens: {
    label: ( obj ) => {
      const color = {
        log: 'cyanBright',
        error: 'red',
        info: 'blue',
        warn: 'yellow',
      }[obj.method];

      return require('chalk')`{${color || 'reset'} ${
        obj.defaultTokens.label( obj )
      }}`;
    },
  },
  stdout: fs.createWriteStream(`./stderr-${new Date().getTime()}.log`),
});
```

**Comment by starak** [comment]
I'll look in to it. Thanks!

**Comment by starak** [comment]
This is not actually a bug. If you look in the readme, it states that `stdout` and `stderr` is for [Custom Console](https://github.com/starak/node-console-stamp/tree/3.0.0#customconsole). But the fix is simple so I'll fix it anyway :)

**Comment by starak** [comment]
Fixed with 3.0.0-rc4.2
